vizro-core/docs/pages/user-guides/assets.md

```diff
@@ -147,42 +155,47 @@ Suppose you want to hide the page title on one page only. Here's how you can ach
 
 
 ## Overwrite CSS properties in selective components
-To overwrite CSS properties of selective components, pass an ID to the relevant component and target the right CSS property.
+To adjust CSS properties for specific components, like altering the appearance of a selected [Card][vizro.models.Card] rather than all Cards,
+you'll need to target the correct CSS selector based on the component hierarchy.
+If you're unfamiliar with CSS selectors, you can refer to this [tutorial](https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_selectors/Selector_structure) for guidance.
 
-For more information, see this [CSS selectors tutorial](https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_selectors/Selector_structure).
+Here's how you can do it:
 
-Let's say we want to change the background and font-color of one [`Card`][vizro.models.Card] instead of all existing Cards in the Dashboard.
-We can use the ID of the outermost Div to target the inner sub-components of the card. Note that all our components have an ID attached to the outermost Div,
-following the pattern `"{component_id}_outer"`.
+1. Assign a unique `id` to the relevant `Card`, for example: `Card(id="custom-card", ...)`
+2. Review the component's source code to find the appropriate CSS class or element you need to target.
```

Comment:
I think this needs a bit more explanation even if it's just a link to a tutorial or an example of how to do it in Chrome (maybe https://stackoverflow.com/questions/4500572/how-can-i-get-the-css-selector-in-chrome?). Currently it's not clear that you should even look at the source code of the page in your browser rather than the Python source code for `vm.Card`.

Reply:
Or are you actually recommending that the user looks at our Python source? Now I'm not even sure 😅

Reply:
Take a look now and let me know what you think: https://github.com/mckinsey/vizro/pull/462/commits/997184fd2da64054e39b4bb598323fe1b502e1e0

A more thorough explanation on how to inspect the console with the Chrome devtools  etc. I would leave for a separate PR